Intervalos/src/LM/intervalos.jl

```diff
@@ -0,0 +1,293 @@
+        infimo=min(prevfloat(a*in_b),prevfloat(a*su_b))
+        supremo=max(nextfloat(a*in_b),nextfloat(a*su_b))
+        return Intervalo(infimo,supremo)
+    else
+        in_a=getfield(a, :infimo)
+        su_a=getfield(a, :supremo)
+        
+        if isnan(prevfloat(in_a*in_b)) && isnan(prevfloat(su_a*su_b))
+            infimo=0.0
+            supremo=0.0
+        elseif isnan(prevfloat(in_a*in_b))
+            infimo=min(0,prevfloat(in_a*su_b),prevfloat(su_a*in_b),prevfloat(su_a*su_b))
+            supremo=max(0,nextfloat(in_a*su_b),nextfloat(su_a*in_b),nextfloat(su_a*su_b))
+        elseif isnan(prevfloat(in_a*su_b))
+            infimo=min(prevfloat(in_a*in_b),0,prevfloat(su_a*in_b),prevfloat(su_a*su_b))
+            supremo=max(nextfloat(in_a*in_b),0,nextfloat(su_a*in_b),nextfloat(su_a*su_b))
+        elseif isnan(prevfloat(su_a*in_b))
+            infimo=min(prevfloat(in_a*in_b),prevfloat(in_a*su_b),0,prevfloat(su_a*su_b))
+            supremo=max(nextfloat(in_a*in_b),nextfloat(in_a*su_b),0,nextfloat(su_a*su_b))
+        elseif isnan(prevfloat(su_a*su_b))
+            infimo=min(prevfloat(in_a*in_b),prevfloat(in_a*su_b),prevfloat(su_a*in_b),0)
+            supremo=max(nextfloat(in_a*in_b),nextfloat(in_a*su_b),nextfloat(su_a*in_b),0)
+        else 
+            infimo=min(prevfloat(in_a*in_b),prevfloat(in_a*su_b),prevfloat(su_a*in_b),prevfloat(su_a*su_b))
+            supremo=max(nextfloat(in_a*in_b),nextfloat(in_a*su_b),nextfloat(su_a*in_b),nextfloat(su_a*su_b))
+        end
+        return Intervalo(infimo,supremo)
+    end
+end
+
+*(a::Intervalo,b::Real)=*(b,a)
+
+import Base: /
+
+function /(a,b::Intervalo)
+    in_b=getfield(b, :infimo)
+    su_b=getfield(b, :supremo)
+    
+    if 0 ∈ b
+        return Intervalo(-Inf, Inf)    
+    elseif typeof(a)<:Real
+        infimo=min(prevfloat(a/in_b),prevfloat(a/su_b))
+        supremo=max(nextfloat(a/in_b),nextfloat(a/su_b))
+        return Intervalo(infimo,supremo)
+    else
+        in_a=getfield(a, :infimo)
+        su_a=getfield(a, :supremo)
+        
+        infimo=min(prevfloat(in_a/in_b),prevfloat(in_a/su_b),prevfloat(su_a/in_b),prevfloat(su_a/su_b))
+        supremo=max(nextfloat(in_a/in_b),nextfloat(in_a/su_b),nextfloat(su_a/in_b),nextfloat(su_a/su_b))
+        return Intervalo(infimo,supremo)
+    end
+end
+
+# Definimos la función para las potencias enteras
+
+import Base: ^
+
+function ^(a::Intervalo,b::Int64)
+    if b==0
+        return Intervalo(1,1)
+    end
+    
+    if isnan(getfield(a, :infimo))
+        return a
+    end
+    
+    a_in=prevfloat(getfield(a,:infimo)^b)
+    a_su=nextfloat(getfield(a,:supremo)^b)
+    
+    if b%2!=0
+        return Intervalo(a_in,a_su)
+    else
+        if getfield(a,:infimo) ≤ 0 && getfield(a,:supremo) ≥ 0
+            supremo= max(a_in,a_su)
+            return Intervalo(0,supremo)
+        else 
+            infimo=min(a_in,a_su)
+            supremo=max(a_in,a_su)
+            return (infimo,supremo)
+        end
+    end
```

Comment:
```suggestion
    a_in=getfield(a,:infimo)^b
    a_su=getfield(a,:supremo)^b

    if b%2!=0
        return Intervalo(prevfloat(a_in),nextfloat(a_su))
    else
        if getfield(a,:infimo) ≤ 0 && getfield(a,:supremo) ≥ 0
            supremo= max(a_in,a_su)
            return Intervalo(0,nextfloat(supremo))
        else 
            infimo=prevfloat(min(a_in,a_su))
            supremo=nextfloat(max(a_in,a_su))
            return (infimo,supremo)
        end
    end
```
Nota el cambio realizado, los next y prev float se realizan cerca del return. No antes... esto es porque para un metodo si servía pero otro te estaba dando problemas.